v4/range.go

```diff
@@ -381,6 +381,12 @@ func expandWildcardVersion(parts [][]string) ([][]string, error) {
 	return expandedParts, nil
 }
 
+// containsWildcard returns true iff there's a wildcard in any of the major, minor or patch components
```

Comment:
```suggestion
// containsWildcard returns true if there's a wildcard in any of the major, minor or patch components
```

Reply:
I meant to express "if and only if" through the [conventional abbreviation _iff_](https://en.wikipedia.org/wiki/If_and_only_if).

Reply:
Some people might be not aware of this. I personally have seen usage of it just a couple of times over last 10 years. But it's up to you.